nav2_map_server/test/component/test_map_saver_2d_node.cpp

```diff
@@ -18,11 +18,15 @@
 #include <rclcpp/rclcpp.hpp>
 #include <string>
 #include <memory>
+#include <vector>
```

Comment:
Unnecessary/unused header. It looks I've already thousand times mentioned about unnecessary headers in other places. Please check the change for all the rest of testing files and remove unnecessary headers from `*.cpp/*.hpp` and unnecessary dependencies from CMakeLists.txt